src/app/src/main/kotlin/org/vaccineimpact/api/app/ErrorHandler.kt

```diff
@@ -26,17 +28,38 @@ open class ErrorHandler(private val serializer: Serializer = MontaguSerializer.i
         sparkException<Exception>(this::handleUnexpectedError)
```

Comment:
Could you now get rid of this line, given that the `handleError` function can now handle non-Montagu exceptions?